bitnami/wordpress/templates/ingress.yaml

```diff
@@ -47,6 +59,9 @@ spec:
     {{- if .Values.ingress.tls }}
     - hosts:
         - {{ .Values.ingress.hostname }}
+        {{- range .Values.ingress.alternativeHostNames }}
```

Comment:
Why don't we use `extraHosts` directly? we would avoid introducing a new value.

Reply:
Good spot. Let me try that